build/package/scripts/build-typescript.sh

```diff
@@ -56,8 +62,24 @@ if [ -n "${NEXUS_HOST}" ] && [ -n "${NEXUS_USERNAME}" ] && [ -n "${NEXUS_PASSWOR
     npm config set strict-ssl=false
 fi;
 
-echo "Building ..."
+echo "Installing dependencies..."
 npm ci
+
+echo "Linting..."
+set +e
+npm run lint > eslint-report.txt
+exitcode=$?
+set -e
+
+if [ $exitcode == 0 ]; then
+  echo "No linting issues inside workspace" > eslint-report.txt
```

Comment:
I think I would prefer a simple `OK`. I need to add this for Go as well, which is also silent on success.